docs/changelog/index.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2024-01-16
+
+* Added page about [running tests](../development/testing.md)
+* Published
+  [tests results as part of Task 6. Base tests](../test-results/2024_01_11_orange_crab_without_create_primary.html)
+* TBD
```

Comment:
@krystian-hebel Would you like to add something more?

Reply:
`Small changes to [description of interrupts handling](../development/soc_fpga_communication.md#interrupts)`

Reply:
1540be7ac8da1210a1a9e2496df163f6ecfe944e